src/coreComponents/constitutive/fluid/CO2BrineFluid.hpp

```diff
@@ -576,23 +574,19 @@ CO2BrineFluid< PHASE1, PHASE2, FLASH >::KernelWrapper::
 ENUM_STRINGS( CO2BrinePhillipsFluid::SubModelInputNames,
               "DensityFun",
               "ViscosityFun",
-              "EnthalpyFun",
-              "InternalEnergyFun" );
+              "EnthalpyFun" );
```

Comment:
I would encourage you to build the same tests that I did in #1778 so future refactorings will be safer.

Reply:
Hmm, I missed these in the PR where they were introduced. We definitely shouldn't be redeclaring these strings 4 times - the enumeration type must be made non-dependent on template parameters of `CO2BrineFluid` in one of the following ways:
  * moving the enum definition out of class into the namespace `geosx::constitutive` (I'd prefer not to)
  * making a non-template class `CO2BrineFluidBase` containing the enum and everything else that does not depend on the template parameters
  * or moving the enum definition entirely into `.cpp` file - it is not used anywhere else. Or maybe get rid if enum entirely? It's not actually used anywhere, except for getting strings (once per value). Or at least it needs to be used as an enum - by converting the input string to it, then switching on the value, instead of branching on string comparisons.

This is a future TODO item, not something to address in current PR, which is ready to be merged.

Reply:
Ok sounds good, I did not know. I am going to merge this PR now since I realize now that it is at the top of the queue, and I will remove these enums immediately in the main thermal PR. Thanks!